java/README.md

```diff
@@ -41,6 +41,7 @@ In the `src/main` directory of the project, you will find several sub-directorie
    2. Optional Parameters:
        * `TOPIC`: Specify the topic for which you wish to publish/subscribe. 
        * `NUMBER_OF_EVENTS_TO_PUBLISH`: Specify the number of events to publish while using the PublishStream RPC.
+       * `SINGLE_BATCH`: Specify if you wish to publish the events in a single or multiple batches, i.e., in separate PublishRequests.
```

Comment:
I removed "in separate" from "in separate PublishRequests" because we're trying to explain what a batch is. Also, changed "wish" to "want" because it is more standard text.

> Specify if you want to publish the events in a single or multiple batches, i.e. PublishRequests.